docker/install-deps

```diff
@@ -127,7 +127,7 @@ if [ -f /usr/bin/apt-get ]; then
         curl bison cmake perl libssl-dev gcc g++ libaio-dev libldap2-dev libwrap0-dev gdb unzip gawk \
         libmecab-dev libncurses5-dev libreadline-dev libpam-dev zlib1g-dev libcurl4-openssl-dev \
         libnuma-dev libjemalloc-dev libc6-dbg valgrind libjson-perl python-mysqldb libevent-dev pkg-config \
-        libmecab2 mecab mecab-ipadic git autoconf libgsasl7 libsasl2-dev libsasl2-modules devscripts \
+        libmecab2 mecab mecab-ipadic git autoconf libgsasl7 libsasl2-dev libsasl2-modules devscripts libeditline-dev \
```

Comment:
@hors Not sure about CentOS/ RHEL but I also tested this on Ubuntu Eoan and got the following results:

1. When we specify `-DWITH_EDITLINE=system` cmake option, we need `libedit-dev`
2. When we specify `-DWITH_READLINE=system` cmake option, we need `libreadline-dev`
3. When we specify `-DWITH_EDITLINE=bundled` cmake option, no additional package is needed
4. When we specify `-DWITH_SYSTEM_LIBS=ON` and no `-DWITH_EDITLINE` / `-DWITH_READLINE`, this implicitly means `-DWITH_READLINE=system`
5. When we specify neither `-DWITH_SYSTEM_LIBS` nor `-DWITH_EDITLINE` / `-DWITH_READLINE`, this implicitly means `-DWITH_EDITLINE=bundled`

In other words, `libeditline-dev` is never required by Percona Server.
According to @EvgeniyPatlan release packages are built with `-DWITH_READLINE=system` and therefore have `libreadline` package as a dependency.

Reply:
I tried to verify how I got that error originally, but I can't reproduce it now :( I've got the same result, all options build without libedit-dev. It's possible that this was an issue with the 8.0.18 merge I've fixed in the meantime.